### PR TITLE
Add initial README

### PR DESCRIPTION
**Pull request by GVarenhorst**
Initial README detailing

Check-ins
Workflow
Project Organization
Code Design (currently missing, will add vs code convention and class types)
DTR document (initial)
Contributors (missing Matthew)

Updated with missing info in code design and contributors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
...
+
+For Project Managers we chose Github Projects as we found it was potentially more versatile than other manager programs and directly integrated with GitHub. We researched Trello as well but found that some of its features required purchase for full usability and GitHub projects offered better ease of communication between team members. Ashley was chosen to host the repo for this project. Retro was planned for 04/25/2023 at 9 AM MT.
+
+  ###Code Design
+
+We established shared conventions for VS Code. We would use double quotes (“”). We would have a vertical attr_reader. Hashes <> Array is expected versus .to eq(hash).
+
+  ###DTR Document
+https://docs.google.com/document/d/1aQTaoqGz3xtM8JAw5DZGTMAQPFfMce8zIxMc6ZQXAz8/edit?usp=sharing
+Initial Document created 04/23/2023
```
Comment (goldenll) [inline README.md]: The document was created on 04/17/2023. Can you please update this date?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,43 @@
 # Futbol
 
 Starter repository for the [Turing School](https://turing.io/) Futbol project.
+
+#README
```
Comment (goldenll) [inline README.md]: When establishing headers, we want to put a space between the # and the word in order to create the header. Without a space, it does not create the header and inserts the "#" into the file

File: README.md [hunk 3]
```diff
@@ -1,3 +1,43 @@
...
+
+	###Check-Ins
```
Comment (goldenll) [inline README.md]: Space between # and word

File: README.md [hunk 4]
```diff
@@ -1,3 +1,43 @@
...
+
+  We had async check-ins in the morning including status updates, with a requested response by 9 AM MT. If working in the evening, Slack status updates before signing off for the next morning are acceptable.
+Live group check-ins during class group time. Stand up responses state the status of assigned tasks; what the programmer will be working on that day; and questions, comments and concerns that inform the group.
+
+  ###Workflow
```
Comment (goldenll) [inline README.md]: Space between # and word

File: README.md [hunk 5]
```diff
@@ -1,3 +1,43 @@
...
+
+Team members assign tasks to themselves, changing status to “In Progress” and creating a pull request when complete, changing status again to “Pull Request”. Pull request workflow is submitted when ready to merge or if stuck or need help, followed by a slack to the group detailing and documenting further any issues. A team member will respond to claim ownership of the review, and reviewers are designated by morning and evening, with Garrett and Ashley reviewing in the AM and Matthew and Lauren taking the PM review shifts. The timeframe for reviewing is up to 12 hours after pull request submission.  Project Board:  https://github.com/users/aplasket/projects/4
+
+###Project Organization
```
Comment (goldenll) [inline README.md]: Space between # and word

File: README.md [hunk 6]
```diff
@@ -1,3 +1,43 @@
...
+
+For Project Managers we chose Github Projects as we found it was potentially more versatile than other manager programs and directly integrated with GitHub. We researched Trello as well but found that some of its features required purchase for full usability and GitHub projects offered better ease of communication between team members. Ashley was chosen to host the repo for this project. Retro was planned for 04/25/2023 at 9 AM MT.
+
+  ###Code Design
```
Comment (goldenll) [inline README.md]: Space between # and word

File: README.md [hunk 7]
```diff
@@ -1,3 +1,43 @@
...
+
+We established shared conventions for VS Code. We would use double quotes (“”). We would have a vertical attr_reader. Hashes <> Array is expected versus .to eq(hash).
+
+  ###DTR Document
```
Comment (goldenll) [inline README.md]: Space between # and word

File: README.md [hunk 8]
```diff
@@ -1,3 +1,43 @@
...
+https://docs.google.com/document/d/1aQTaoqGz3xtM8JAw5DZGTMAQPFfMce8zIxMc6ZQXAz8/edit?usp=sharing
+Initial Document created 04/23/2023
+
+
+
+   ##CONTRIBUTORS
```
Comment (goldenll) [inline README.md]: Space between # and word

**Review by goldenll — COMMENTED** [review]
The content looks good! We just need to make some minor edits before this will be ready to push to main. Please see comments

File: README.md [hunk 9]
```diff
@@ -1,3 +1,43 @@
...
+
+Ashley Plasket  	    Github: https://github.com/aplasket
```
Comment (goldenll) [inline README.md]: We may also want to edit this section for readability. Double return at the end of the line will create a line break. We may also want to use bullet points for the GitHub and LinkedIn links?

File: README.md (hunk 4 above)
Comment (MWMJohnson) [reply README.md]: The indentation on lines 9 and 10 looks off in the editor. Can you please adjust?

File: README.md (hunk 7 above)
Comment (MWMJohnson) [reply README.md]: maybe line 22 should be in bulleted format.

File: README.md (hunk 5 above)
Comment (MWMJohnson) [reply README.md]: maybe break this paragraph up.

File: README.md (hunk 1 above)
Comment (GVarenhorst) [reply README.md]: Updated date to 04/17/2023

File: README.md (hunk 2 above)
Comment (GVarenhorst) [reply README.md]: Space added

File: README.md (hunk 3 above)
Comment (GVarenhorst) [reply README.md]: Space added

File: README.md (hunk 4 above)
Comment (GVarenhorst) [reply README.md]: Corrected space and indention

File: README.md (hunk 5 above)
Comment (GVarenhorst) [reply README.md]: added space

File: README.md (hunk 6 above)
Comment (GVarenhorst) [reply README.md]: Added space

File: README.md (hunk 7 above)
Comment (GVarenhorst) [reply README.md]: Added space, updated Code Design

File: README.md (hunk 8 above)
Comment (GVarenhorst) [reply README.md]: Added space

File: README.md (hunk 9 above)
Comment (GVarenhorst) [reply README.md]: added space, reformatted Contributors

**Pull request merged**